rns_shell/platform/graphics/libwpe/PlatformDisplayLibWPE.h

```diff
@@ -21,15 +21,15 @@ class Display {
 public:
     struct wpe_view_backend* viewBackend() const { return viewBackend_; }
```

Comment:
do we  need the viewBackend_ in this class now ?

Reply:
Good Catch. Not Needed ,it's a dead code now. will remove it.